hooks/scheduler.js

```diff
@@ -92,11 +97,12 @@ class Scheduler {
 
   /** Handle spawning a new task for a given hook that needs to be scheduled */
   async handleHook(hook) {
-    let task = await hook.taskPayload();
-    let resp = await queue.createTask(hook.nextTaskId, task);
+    // TODO: (when we have hook logging) if this fails due to 401, we should
+    // still consider it scheduled
+    await this.taskcreator.fire(hook, {}, {taskId: hook.nextTaskId});
```

Comment:
provide `created` see other comments on it:
`await this.taskcreator.fire(hook, {}, {taskId: hook.nextTaskId, created: hook.nextScheduledDate});`